test/common/ops/reset.js

```diff
@@ -52,6 +52,20 @@ describe('shared.ops.reset', () => {
     expect(user.stats.lvl).to.equal(1);
   });
 
+  it('resets user\'s stat points (str, con, int, per)', () => {
+    user.stats.str = 2;
+    user.stats.con = 2;
+    user.stats.int = 2;
+    user.stats.per = 2;
```

Comment:
Could you please add stats.points to this and to the expect() statements below? :)

In case you're not aware, you can make another commit on the same branch in your local install and push it to your github fork, and it will automatically appear in this PR.

Reply:
Thank you for your support and encouragement, Alys! You are super!